src/tasks/readme.js

```diff
@@ -4,14 +4,28 @@ const { json, template } = require('mrm-core');
 module.exports = () => {
   const pkg = json('package.json');
 
-  // Create Readme.md (no update)
+  // Create README.md (no update)
   const readme = template('README.md', path.join(__dirname, '../../templates/README.template'));
+
   if (!readme.exists()) {
-    readme
-      .apply({
-        package: pkg.get('name'),
-        description: pkg.get('description') || 'DESCRIPTION',
-      })
-      .save();
+    const getCase = name => name.replace(
```

Comment:
I don't use `lodash`, therefore I didn't know `_.startCase` existed 😛 I interpreted your earlier comment as 'create helper fn' for casing :D. I'll take a look at `_.startCase` later today :ghost: